doc/developer-guides/hld/ivshmem-hld.rst

```diff
@@ -181,3 +181,57 @@ post-launched VMs (VM1 and VM2).
+   - Don't implement your own crypto and use available Intel crypto libraries when applicable or other open source complaint libraries. (`Intel IPP <https://github.com/intel/ipp-crypto>`_, `TinyCrypt <https://01.org/tinycrypt>`_)
+   - Utilize the platform/kernel infrastructure and services (e.g. `vTPM <https://projectacrn.github.io/latest/developer-guides/hld/hld-security.html?highlight=vtpm>`_, `Kernel Crypto backend/APIs <https://www.kernel.org/doc/html/v5.4/crypto/index.html>`_ , `keyring subsystem <https://www.man7.org/linux/man-pages/man7/keyrings.7.html>`_, etc..).
+   - Depending on the used crypto key type and in case of requirements for key persistence across system/power management events, implement any necessary flows for key lifecycle management including wrapping, revocation and migration.
+   - Follow open source secure crypto coding guidelines for secure wrappers and marshalling data structures: `Secure Applied Crypto <https://github.com/veorq/cryptocoding>`_
+   - References: `NIST Crypto Standards and Guidelines <https://csrc.nist.gov/projects/cryptographic-standards-and-guidelines>`_, `OpenSSL <https://www.openssl.org/>`_
+
+
+4. **Applications Whitelisting**
+
+   - For use cases implemented in static environments for (Ex: Industrial/Automotive usages, make sure to follow application whitelist techniques and disable any 3rd party or native app stores.
+   - This mechanism can be chained with the access control policies to protect access to whitelisting rules/configuration files. Please refer to opensource or implement your custom solution.
+   - References: `NIST SP800-167 <https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-167.pdf>`_, `fapolicyd <https://github.com/linux-application-whitelisting/fapolicyd>`_
+
+
+5. **Secure boot and FS Integrity Verification**
+
+   - All the previous highlighted technologies mostly relays on the kernel as a secure component to enforce such policies. As a result, it is strongly recommended to enable secure boot for the SOS and extend the secure boot chain to Post-launch VMs kernel.
+   - To ensure no malicious software is introduced or persists, utilize the file system verification methods on every boot to extend the secure boot chain for post-launch VMs (kernel/FS).
+   - Reference: ACRN secure boot extension guide (`ClearLinux <https://projectacrn.github.io/latest/tutorials/enable_laag_secure_boot.html?highlight=secure%20boot>`_, `Windows <https://projectacrn.github.io/latest/tutorials/waag-secure-boot.html>`_)
+   - Reference Stack:  `dm-verity <https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/verity.html>`_
+
+.. note:: All the mentioned hardening techniques might require minor extra development efforts.`
```

Comment:
Delete the trailing backtick on this line

Reply:
done